app/src/main/AndroidManifest.xml

```diff
@@ -4,6 +4,7 @@
 
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
```

Comment:
Why do we need location?

Reply:
Ah! Will remove it. We don't need to access GPS. Thanks